packages/spec/jest.setup.js

```diff
@@ -1 +1,20 @@
 jest.setTimeout(5 * 60 * 1000);
+
+const isReactLifecycleWarning = (warning) => {
+  const reReactLifecycleWarning = new RegExp(
+    'Warning: componentWill[^\\s]+ has been renamed, and is not recommended for use. ' +
+      'See https://fb.me/react-async-component-lifecycle-hooks for details.'
+  );
+  return Boolean(warning.match(reReactLifecycleWarning));
+};
+
+const isIntolerableWarning = (type, text) =>
+  type === 'warning' && !isReactLifecycleWarning(text);
+
+global.handleConsoleOutput = (msg) => {
```

Comment:
Not sure why, but for some reason adding a global here seems different to me than the globals that the Jest environment exposes and I'd prefer it not be a global, but I can't quite put my finger on why it it's different..

Reply:
At the end of the day the globals, exposed by the Jest env, suck, too, in a way. 🤷‍♂️ 

We can export this function from a `helper.js` in the `spec`-folder. It doesn't need to be global.

Reply:
After thinking about this a bit more, I think the reason why this doesn't sit well with me is:
Providing globals through the Jest environment is part of their API contract (provided by `this.globals` instead of `global` - the Node global environment) and is considered acceptable (see `describe`, `it` and `expect` functions).
Furthermore: We have documented them in the package and on can easily find them by looking up the environment.
Once we start adding more globals from different places (such as a `jest.setup.js` file), we distribute the places where one has to look them up.
But as said in my comment above, this is mostly hairsplitting and probably some gut feeling.

Reply:
Fine for me. So what do you think about importing the function from a helper file instead?

Reply:
Yes, sounds good :+1: